src/traces/indicator/index.js

```diff
@@ -0,0 +1,30 @@
+/**
+* Copyright 2012-2019, Plotly, Inc.
+* All rights reserved.
+*
+* This source code is licensed under the MIT license found in the
+* LICENSE file in the root directory of this source tree.
+*/
+
+'use strict';
+
+module.exports = {
+    moduleType: 'trace',
+    name: 'indicator',
+    basePlotModule: require('./base_plot'),
+    categories: ['svg', 'noOpacity', 'noHover'],
```

Comment:
It could be useful to if this trace had an adjustable opacity.

Reply:
Mojtaba _"Mr. Opacity"_ Samimi